net-analyzer/linkchecker/linkchecker-10.2.1_p20230123.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..11} )
+PYTHON_REQ_USE="sqlite?"
+DISTUTILS_USE_PEP517=hatchling
+DISTUTILS_SINGLE_IMPL=1
+
+inherit bash-completion-r1 distutils-r1 multiprocessing optfeature
+
+DESCRIPTION="Check websites for broken links"
+HOMEPAGE="https://github.com/linkcheck/linkchecker"
+
+if [[ "${PV}" == "9999" ]]; then
+	EGIT_REPO_URI="https://github.com/linkcheck/linkchecker.git"
+	inherit git-r3
+else
+	COMMIT="3041c8af2836bb3b3388ab8ad28f0d32bfbbbc26"
+	SRC_URI="https://github.com/linkchecker/linkchecker/archive/${COMMIT}.tar.gz -> ${P}.gh.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+	S="${WORKDIR}/${PN}-${COMMIT}"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="bash-completion nls pdf sqlite"
+
+RDEPEND="
+	$(python_gen_cond_dep '
+		dev-python/beautifulsoup4[${PYTHON_USEDEP}]
+		dev-python/dnspython[${PYTHON_USEDEP}]
+		dev-python/requests[${PYTHON_USEDEP}]
+	')
+	bash-completion? (
+		$(python_gen_cond_dep 'dev-python/argcomplete[${PYTHON_USEDEP}]') )
+	pdf? (
+		$(python_gen_cond_dep 'app-text/pdfminer[${PYTHON_USEDEP}]') )
+"
+BDEPEND="
+	$(python_gen_cond_dep '
+		dev-python/hatch-vcs[${PYTHON_USEDEP}]
+	')
+	nls? (
+		$(python_gen_cond_dep '
+			dev-python/polib[${PYTHON_USEDEP}]
+		')
```

Comment:
> What should we use? 

`optfeature` since they have no function in the ebuild. But I got curious about the `nls` one, is that an automagic condition? It should be fixed.


> I note if we provide explanations in the local metadata.xml (as for sqlite below) `equery u` uses the global descriptions.

That shouldn't happen, descriptions will always prioritize local *metadata.xml* file over global descriptions. It could be your metadata cache hasn't been generated for the tree with this change applied though.